app/src/main/java/in/testpress/testpress/util/RegisterFormUserInputValidation.kt

```diff
@@ -0,0 +1,108 @@
+package `in`.testpress.testpress.util
+
+import `in`.testpress.testpress.databinding.RegisterActivityBinding
+import `in`.testpress.testpress.enums.VerificationMethod
+import android.view.View
+import android.widget.EditText
+import android.widget.TextView
+import androidx.databinding.BaseObservable
+
+class RegisterFormUserInputValidation(
+        private var binding: RegisterActivityBinding,
+        private val verificationMethod: VerificationMethod,
+        private val isTwilioEnabled: Boolean
+) : BaseObservable() {
+
+    private var isValid = true
+    private val registerFormEmptyInputValidation = RegistrationFormEmptyInputValidation(binding,verificationMethod)
```

Comment:
Is this necessary? Can't this class itself check for empty values validation? Isn't this overcomplicating?